components/notebook-controller/config/manager/kustomization.yaml

```diff
@@ -7,3 +7,4 @@ configMapGenerator:
   literals:
   - USE_ISTIO=true
   - ISTIO_GATEWAY=kubeflow/kubeflow-gateway
+  - ISTIO_HOST='*'
```

Comment:
Because the field of the `VirtualService` is called `hosts` (not `host`) I think the variable should be: `ISTIO_HOSTS`

Reply:
Good call out, updated :)